packages/xo-web/src/common/utils.js

```diff
@@ -601,3 +601,18 @@ export const getIscsiPaths = pbd => {
   const pathsInfo = pbd.otherConfig[`mpath-${pbd.device_config.SCSIid}`]
   return pathsInfo !== undefined ? JSON.parse(pathsInfo) : []
 }
+
+// ===================================================================
+
+export const downloadLog = ({ log, date, type }) => {
+  const file = new window.Blob([log], {
+    type: 'text/plain',
+  })
+  const anchor = document.createElement('a')
+  anchor.href = window.URL.createObjectURL(file)
+  anchor.download = `${new Date(date).toISOString()} - ${type}.log`
+  anchor.style.display = 'none'
+  document.body.appendChild(anchor)
+  anchor.click()
+  document.body.removeChild(anchor)
```

Comment:
Have you tested it on both Chrome and Firefox?

Reply:
Yes, but only with the last version of both.

Reply:
Good enough.